gradle-mvn-push.gradle

```diff
@@ -32,11 +32,21 @@ def getSnapshotRepositoryUrl() {
 }
 
 def getRepositoryUsername() {
-  return hasProperty('NEXUS_USERNAME') ? NEXUS_USERNAME : ""
```

Comment:
You can use `ORG_GRADLE_PROJECT_NEXUS_USERNAME` to set properties from environment variables and retain the original behavior.